server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaBase.ts

```diff
@@ -26,6 +19,7 @@ export interface IKafkaEndpoints {
 }
 
 export abstract class RdkafkaBase extends EventEmitter {
+    protected readonly kafka: typeof kafkaTypes;
```

Comment:
nit: it looks like this file uses tabs... (yuck)

Reply:
converted to space